diff --git a/node_modules/ngx-electron/lib/electron.service.d.ts b/node_modules/ngx-electron/lib/electron.service.d.ts
index 15841b1..c8e90c3 100644
--- a/node_modules/ngx-electron/lib/electron.service.d.ts
+++ b/node_modules/ngx-electron/lib/electron.service.d.ts
@@ -14,7 +14,6 @@ export declare class ElectronService {
     readonly isArm: boolean;
     readonly desktopCapturer: Electron.DesktopCapturer;
     readonly ipcRenderer: Electron.IpcRenderer;
-    readonly remote: Electron.Remote;
     readonly webFrame: Electron.WebFrame;
     readonly clipboard: Electron.Clipboard;
     readonly crashReporter: Electron.CrashReporter;
